lib/gateway/postcard_api.dart

```diff
@@ -44,8 +44,9 @@ abstract class PostcardApi {
     @Part(name: "counter") required int counter,
   });
 
-  @GET("/v1/leaderboard?unit={unit}")
-  Future<GetLeaderboardResponse> getLeaderboard(@Path("unit") String unit);
+  @GET("/v1/leaderboard?unit={unit}&size={size}&offset={offset}")
+  Future<GetLeaderboardResponse> getLeaderboard(@Path("unit") String unit,
```

Comment:
using `@Query` instead of `@Path` it would be simpler and might avoid doing `?` `&` `=` manually.